lxd/main_forkmount.go

```diff
@@ -152,14 +153,35 @@ void do_lxd_forkmount(pid_t pid) {
 		_exit(1);
 	}
 
+	if (strcmp(shiftfs, "true") == 0) {
```

Comment:
We should really just do that `strcmp()` only once. But I'm not going to block the PR on that now. We can fix that later.